test/smoke_test.rb

```diff
@@ -23,6 +23,8 @@ class MovieSearch < FortyFacets::FacetSearch
   facet [:studio, :country], name: 'Country'
   facet [:studio, :status], name: 'Studio status'
   facet [:studio, :producers], name: 'Producers'
+  scope_facet [:classics, :non_classics], name: 'Classics', path: 'classics'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.